### DISCUSSION TITLE
argo-workflows/faq/

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#10279] [General]
# argo-workflows/faq/

https://argoproj.github.io/argo-workflows/faq/

**Comment by nicolas-vivot** [comment]
Please ADD some FAQ or topic about incompatibilities with Helm
https://github.com/argoproj/argo-workflows/issues/2430

**Comment by maor-paz-hs** [comment]
Having 2 pods of type server of Argo Workflow, 
Getting this error:
16 - UNAUTHENTICATED

What can be the issue? 

Have a theory though it's related to the stickiness of the LoadBalnacer.
